portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -17,16 +17,20 @@
 public final class Comment {
 
   private final long id;
-  private final String text;
   private final long postTime;
+  private final String text;
+  private final String  userEmail;
```

Comment:
```suggestion
  private final String userEmail;
```
nit: remove extra space

Reply:
great catch